test/common.js

```diff
@@ -1,5 +1,6 @@
 process.env.NODE_ENV = 'test';
 process.env.LOSANT_API_URL = process.env.LOSANT_API_URL || 'https://api.losant.space';
+process.env.DEFAULT_API_URL = process.env.DEFAULT_API_URL || 'https://api.losant.space';
```

Comment:
in general,I think I'd prefer *not* to litter our private staging url all over these tests